app/components/subject-picker/component.js

```diff
@@ -158,13 +130,16 @@ export default Component.extend(Analytics, {
                 .slice(0, nextTier)
                 .map(column => column.get('selection'));
 
-            // An existing tag has this prefix, and this is the lowest level of the taxonomy, so no need to fetch child results
-            if (nextTier === totalColumns || !allSelections.some(item => arrayStartsWith(item, currentSelection))) {
+            // An existing tag has this prefix, and this is the lowest level of the taxonomy,
+            // so no need to fetch child results
+            if (nextTier === totalColumns
+                || !allSelections.some(item => arrayStartsWith(item, currentSelection))) {
                 let existingParent;
 
                 for (let i = 1; i <= currentSelection.length; i++) {
                     const sub = currentSelection.slice(0, i);
-                    existingParent = allSelections.find(item => arrayEquals(item, sub)); // jshint ignore:line
+                    existingParent = allSelections
+                        .find(item => arrayEquals(item, sub)); // jshint ignore:line
```

Comment:
check if that comment can be removed

Reply:
It can be 🥇 We shouldn't be using jslint at all anymore